flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -341,8 +341,7 @@ private static Configuration createExecutionConfig(
 				availableCommandLines,
 				activeCommandLine);
 
-		Configuration executionConfig = activeCommandLine.applyCommandLineOptionsToConfiguration(
-				commandLine);
+		Configuration executionConfig = activeCommandLine.toConfiguration(commandLine);
```

Comment:
I am not sure if here now we are missing the options from the base configuration. I will check further but I am also writing it as a comment so that you can also have a look.

Reply:
This works because someone was clever enough to think about this case before. 😅
https://github.com/apache/flink/blame/733c684f19bc9d6167703e977f44d817e3ec5e69/flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java#L192